osu.Game/Online/Spectator/FrameHeader.cs

```diff
@@ -44,28 +44,43 @@ public class FrameHeader
         [Key(4)]
         public DateTimeOffset ReceivedTime { get; set; }
 
+        [Key(5)]
+        public double? BaseScore { get; set; }
+
+        [Key(6)]
+        public double? BonusScore { get; set; }
+
+        [Key(7)]
+        public ScoringValues? Maximum { get; set; }
```

Comment:
I think this should also be called `MaximumScoringValues`? `Maximum` on its own doesn't mean much to me.

Reply:
Yeah I wasn't sure on the naming, felt `Maximum` means enough to signify maximum achievable value, but maybe not. Will go back to `MaximumScoringValues` for now.